test/functional/cmdLineTests/ignoreUnrecognizedVMOptions/ignoreunrecognizedxxcolonoptions.xml

```diff
@@ -0,0 +1,133 @@
+  Exception [1] and GNU General Public License, version 2 with the
+  OpenJDK Assembly Exception [2].
+
+  [1] https://www.gnu.org/software/classpath/license.html
+  [2] http://openjdk.java.net/legal/assembly-exception.html
+
+  SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
+<!DOCTYPE suite SYSTEM "cmdlinetester.dtd">
+
+<suite id="IgnoreUnrecognizedXXColonOptions Command-Line Option Tests" timeout="2400">
+
+ <variable name="IGNOREUNRECOGNIZEDXXOPTIONS" value="-XX:+IgnoreUnrecognizedXXColonOptions" />
+ <variable name="NOIGNOREUNRECOGNIZEDXXOPTIONS" value="-XX:-IgnoreUnrecognizedXXColonOptions" />
+
+ <variable name="BADOPTION1" value="-XX:INVALID" />
+ <variable name="BADOPTION2" value="-XX:-INVALID" />
+ <variable name="BADOPTION3" value="-XX:+INVALID" />
+ <variable name="GOODOPTION" value="-XX:+DisableExplicitGC" />
+
+ <test id="test -XX:+IgnoreUnrecognizedXXColonOptions, bad option: -XX:INVALID">
+  <command>$EXE$ $IGNOREUNRECOGNIZEDXXOPTIONS$ $BADOPTION1$ -version</command>
+  <output type="success" caseSensitive="no" regex="yes" javaUtilPattern="yes">(java|openjdk) version</output>
+  <output type="failure" caseSensitive="no" regex="no">Command-line option unrecognised</output>
+ </test>
+
+ <test id="test -XX:+IgnoreUnrecognizedXXColonOptions, bad option: -XX:-INVALID">
+  <command>$EXE$ $IGNOREUNRECOGNIZEDXXOPTIONS$ $BADOPTION2$ -version</command>
+  <output type="success" caseSensitive="no" regex="yes" javaUtilPattern="yes">(java|openjdk) version</output>
+  <output type="failure" caseSensitive="no" regex="no">Command-line option unrecognised</output>
+ </test>
+
+ <test id="test -XX:+IgnoreUnrecognizedXXColonOptions, bad option: -XX:+INVALID">
+  <command>$EXE$ $IGNOREUNRECOGNIZEDXXOPTIONS$ $BADOPTION3$ -version</command>
+  <output type="success" caseSensitive="no" regex="yes" javaUtilPattern="yes">(java|openjdk) version</output>
+  <output type="failure" caseSensitive="no" regex="no">Command-line option unrecognised</output>
+ </test>
+
+ <test id="test -XX:+IgnoreUnrecognizedXXColonOptions, good option: -XX:+DisableExplicitGC">
+  <command>$EXE$ $IGNOREUNRECOGNIZEDXXOPTIONS$ $GOODOPTION$ -version</command>
+  <output type="success" caseSensitive="no" regex="yes" javaUtilPattern="yes">(java|openjdk) version</output>
+  <output type="failure" caseSensitive="no" regex="no">Command-line option unrecognised</output>
+ </test>
+ 
+ <test id="test -XX:-IgnoreUnrecognizedXXColonOptions, bad option: -XX:INVALID">
+  <command>$EXE$ $NOIGNOREUNRECOGNIZEDXXOPTIONS$ $BADOPTION1$ -version</command>
+  <output type="success" caseSensitive="no" regex="no">Command-line option unrecognised</output>
+  <output type="failure" caseSensitive="no" regex="yes" javaUtilPattern="yes">(java|openjdk) version</output>
+ </test>
+
+ <test id="test -XX:-IgnoreUnrecognizedXXColonOptions, bad option: -XX:-INVALID">
```

Comment:
This and the next are redundant.

Reply:
From this comment, I'll assume that we only need one `BADOPTION`